docs/web-ui.md

```diff
@@ -369,3 +369,43 @@ The web UI includes a Streaming tab if the application uses Spark streaming. Thi
 scheduling delay and processing time for each micro-batch in the data stream, which can be useful
 for troubleshooting the streaming application.
 
+## JDBC/ODBC Server Tab
+We can see this tab when Spark is running as a [distributed sql engine](sql-distributed-sql-engine.html) and shows information about sessions and sql jobs submitted.
+
+At the beginning of the page is the server start time.
+
+<p style="text-align: center;">
+  <img src="img/JDBCServer1.png" width="40%" title="JDBC/ODBC Header" alt="JDBC/ODBC Header">
+</p>
+
+The first section contains information of active and finished sessions with some information.
+* User and IP of the connection.
+* Session id link to access to session info.
+* Start time, finish time and duration of the session.
+* Total jobs submitted in this session.
+
+<p style="text-align: center;">
+  <img src="img/JDBCServer2.png" title="JDBC/ODBC sessions" alt="JDBC/ODBC sessions">
+</p>
+
+The second section has the sql statistics of the submitted  jobs.
+* **User** that submit the job.
+* **Job id** link to [jobs tab](web-ui.html#jobs-tab).
+* **Group id** of the query that group all jobs together. An application can cancel all running jobs using this group id.
+* **Start time** of the process.
+* **Finish time** of the execution, before fetching the results.
+* **Close time** of the process after fetching the results.
+* **Execution time** is the difference between finish time and start time.
+* **Duration time** is the difference between close time and start time.
+* **Statement** is the Sql statement executed.
+* **State** of the process.
+	* _Started_, first state, when the process begins.
+	* _Compiled_, execution plan generated.
+	* _Failed_, final ko state.
```

Comment:
We just added the `CANCELED` state: [SPARK-28901](https://issues.apache.org/jira/browse/SPARK-28901).

Reply:
Thank you i had missed this change